index.js

```diff
@@ -1082,7 +1103,7 @@ module.exports = function (app) {
       }
       if (typeof configuration.bowHeight !== 'undefined') {
         values.push({
-          path: 'design.bowAnchorHight',
+          path: 'design.bowAnchorHeight',
```

Comment:
I don't want to change this because it will break existing apps, including WilhelmSK.

We either need to keep it the same, or push both spellings.

Reply:
Kept both paths and marked misspelt path as deprecated.